_partials/servicesOffer.html.haml

```diff
@@ -0,0 +1,9 @@
+.panel.panel-primary
+  .panel-heading Paid support and consulting
+  .panel-body
+    %p
+      %b Want to talk to the experts?
+      Red Hat offers certified binaries with enterprise consulting.
+      Contact
+      %a.servicesMailAddress optaplanner-info
```

Comment:
Who is monitoring this email address?

Reply:
I am :-) . And Geoffrey. And you can as well. It's a Red Hat mailing list, so you can subscribe to it as well.